docs/drivers.rst

```diff
@@ -136,6 +136,7 @@ Helpers for Bluetooth Low Energy (BLE).
     BLE Adafruit Services <https://circuitpython.readthedocs.io/projects/ble_adafruit/en/latest/>
     BLE Apple Media Service <https://circuitpython.readthedocs.io/projects/ble_apple_media/en/latest/>
     BLE Apple Notification Center Service <https://circuitpython.readthedocs.io/projects/ble_apple_notification_center/en/latest/>
+    BLE Location Beacons <https://circuitpython.readthedocs.io/projects/ble_apple_notification_center/en/latest/>
```

Comment:
This is the wrong URL; it’s a duplicate of the one above.

Reply:
Ha!  Good catch.